### ISSUE TITLE
How to focus filter and item select

### ISSUE THREAD

**Issue by lovelylip** [frapontillo/angular-bootstrap-duallistbox#26]
Hello everybody, I want to focus and  add tabindex in both filter, button, and each option of select, and hotkey 'ENTER' in select1 can move data to select2, replace change or click with keycode = 1, and execute with 'LEFT' & 'RIGHT' move via tabIndex. Help me. Thank all

**Comment by sadiksha** [comment]
Yes, I also need some functionality like this. I tried setting move-on-select to false. But, it didn't solve the issue as I had to select an item and click on the -> button. This is not optimal for people who cannot use mouse and have to use keyboards.
